packages/docs/src/Menu.tsx

```diff
@@ -6,19 +6,41 @@ import {
   ExpandableListItemType,
   IconButton,
 } from 'practical-react-components-core'
-import { HamburgerMenuIcon } from 'practical-react-components-icons'
+import {
+  HamburgerMenuIcon,
+  HomeIcon,
+  LogIcon,
+} from 'practical-react-components-icons'
 
 import { Components, Component } from './types'
 
+const GithubIcon = React.memo((props: React.SVGProps<SVGSVGElement>) => (
+  // Borrowed from Wikipedia
```

Comment:
It's MIT license, I think this has to be included somewhere.

Reply:
Sure. How / where?